README.md

```diff
@@ -28,6 +28,11 @@ trigger when going through the code samples. We hope to improve on this via
 [speaker notes](https://github.com/google/comprehensive-rust/issues/53) and by
 [publishing videos](https://github.com/google/comprehensive-rust/issues/52).
 
+## Press
```

Comment:
I think "Press" is fine, if a tiny bit cheeky since you wrote the press. This is a place someone might look if they wanted to see how to publicize this on their socials.

Reply:
🙂  Maybe "News" would be better?

Reply:
Let me go with Press for now, then we can change it later as we add more here.